assets/src/edit-story/components/library/panes/media/mediaElement.js

```diff
@@ -190,6 +191,7 @@ const MediaElement = ({
           ref={mediaElement}
           width={width}
           height={height}
+          alt={alt}
```

Comment:
This might very well still be empty though (all uploads made in the editor will have empty alt as far as I can tell).

Could we perhaps put the image file name there in that case? I think it's better than nothing.

Reply:
Although we have `alt: alt || description,` - it's not enough (description is empty for fresh files).

So maybe https://github.com/google/web-stories-wp/issues/1092#issuecomment-610882919

Reply:
I know could be empty, but for now I think this is a good work around. Longer term, I would like a better default, but I don't know what this should be.

Reply:
We have the file name in the API. Should be simple enough to change to 
``alt: alt || description || filename``

Reply:
If you could change it like this, that would be great 👍

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/1101/commits/2a336b9a6bab579ebca23924f92c316bafab392b @swissspidy